stable/airflow/CHANGELOG.md

```diff
@@ -0,0 +1,197 @@
+# Changelog
+
+## v7.1.0
+
+The following IMPROVEMENTS have been made:
+* We have dramatically reduced the start time of airflow pods.
+  This was mostly achieved by removing arbitrary delays in the start commands for airflow pods.
```

Comment:
I do not remember why they were needed, do you know?

Reply:
I think it was a bad attempt at forcing the pods to wait for dependencies, but fundamentally, thats not how K8S works, when we can just have them retry until the dependency is available, as not to increase the time under normal operation.